buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -382,8 +384,8 @@ class BeamModulePlugin implements Plugin<Project> {
     def netty_version = "4.1.30.Final"
     def postgres_version = "42.2.2"
     def powermock_version = "2.0.2"
-    def proto_google_common_protos_version = "1.12.0"
-    def protobuf_version = "3.6.0"
+    def proto_google_common_protos_version = "1.16.0"
+    def protobuf_version = "3.10.0"
```

Comment:
`Proto-google-cloud-pubsub-v1:1.75.0` uses google-common-protos 1.16.0 (which has `com.google.api.ClientProto` class). It also uses protobuf 3.10.0.
https://search.maven.org/artifact/com.google.api.grpc/proto-google-cloud-pubsub-v1/1.75.0/jar